docs/source/tutorial.rst

```diff
@@ -117,39 +123,73 @@ build, run the following inside vagrant:
 
   $ /project/sawtooth-core/bin/build_all
 
-Running validator
-=================
+Validator Start-up Process
+==========================
+
+Create Genesis Block
+--------------------
+
+The first time that a validator is run, the validator needs to ingest a batch
+of transactions that set up some initial values for use by the first block in
+the block-chain, called the "genesis block."
+
+To create the genesis block, log in to the development environment with the
+command `vagrant ssh` and run the following command:
+
+.. code-block:: console
+
+  $ sawtooth admin genesis
```

Comment:
@dcmiddle I slightly disagree. I think that should be a separate tutorial, because it is more production oriented, and more complex. The intent of this tutorial is to make it as simple as possible to try out a few of the basic features and get a validator running. I'll ping you on slack over this.